narwhals/dataframe.py

```diff
@@ -215,6 +217,24 @@ class DataFrame(BaseFrame):
         │ 1   ┆ 3   │
         │ 2   ┆ 4   │
         └─────┴─────┘
+
+
+        >>> # Using Pandas Dataframe
```

Comment:
you shouldn't leave this as a comment, you can just write it:

```suggestion
        Using Pandas Dataframe
```